docs/first_objective.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>Can you output these series/dataframes? It will be nice to see the processed data before plotting it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>This is done, added extra cells to print the dataframes</p>